.github/workflows/submit-gradle-dependencies.yml

```diff
@@ -20,5 +20,5 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           dependency-graph: generate-and-submit
-      - name: Generate the dependency graph which will be submitted post-job
-        run: ./gradlew :WordPress:dependencies --no-configure-on-demand
```

Comment:
`--no-configure-on-demand` option is already set by the v3 version of this GH Action. I tested it on my fork: https://github.com/wzieba/WordPress-Android/actions/runs/9856525171/job/27213706734#step:6:36

<img width="1808" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/5845095/fbdf0247-9c42-4c49-ae82-2a022d72804f">

Reply:
Awesome, thanks for testing that on your fork! 🥇